src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -86,53 +86,14 @@ class PropertyAccessor implements PropertyAccessorInterface
      * Should not be used by application code. Use
      * {@link PropertyAccess::createPropertyAccessor()} instead.
      *
-     * @param int                                 $magicMethods       A bitwise combination of the MAGIC_* constants
-     *                                                                to specify the allowed magic methods (__get, __set, __call)
-     *                                                                or self::DISALLOW_MAGIC_METHODS for none
-     * @param int                                 $throw              A bitwise combination of the THROW_* constants
-     *                                                                to specify when exceptions should be thrown
-     * @param PropertyReadInfoExtractorInterface  $readInfoExtractor
-     * @param PropertyWriteInfoExtractorInterface $writeInfoExtractor
+     * @param int $magicMethods A bitwise combination of the MAGIC_* constants
+     *                          to specify the allowed magic methods (__get, __set, __call)
+     *                          or self::DISALLOW_MAGIC_METHODS for none
+     * @param int $throw        A bitwise combination of the THROW_* constants
+     *                          to specify when exceptions should be thrown
      */
-    public function __construct(/*int */$magicMethods = self::MAGIC_GET | self::MAGIC_SET, /*int */$throw = self::THROW_ON_INVALID_PROPERTY_PATH, CacheItemPoolInterface $cacheItemPool = null, /*PropertyReadInfoExtractorInterface */$readInfoExtractor = null, /*PropertyWriteInfoExtractorInterface */$writeInfoExtractor = null)
+    public function __construct(int $magicMethods = self::MAGIC_GET | self::MAGIC_SET, int $throw = self::THROW_ON_INVALID_PROPERTY_PATH, CacheItemPoolInterface $cacheItemPool = null, PropertyReadInfoExtractorInterface $readInfoExtractor = null, PropertyWriteInfoExtractorInterface $writeInfoExtractor = null)
     {
-        if (\is_bool($magicMethods)) {
-            trigger_deprecation('symfony/property-access', '5.2', 'Passing a boolean as the first argument to "%s()" is deprecated. Pass a combination of bitwise flags instead (i.e an integer).', __METHOD__);
```

Comment:
You can remove `symfony/deprecation-contracts` from the composer.json file. See https://github.com/symfony/symfony/pull/41298#discussion_r635621710